output.go

```diff
@@ -24,6 +25,79 @@ Txout-script / scriptPubKey   Script                                      <out-s
 type Output struct {
 	Satoshis      uint64
 	LockingScript *bscript.Script
+	index         int
```

Comment:
@theflyingcodr yo i just realised this, don't think this should be in here... do you?

Reply:
I think I was using this to track the vout, I'll need to double the code again as it's been a while!